Library/ViewModels/NoShippingPledgeViewModel.swift

```diff
@@ -521,7 +521,8 @@ public class NoShippingPledgeViewModel: NoShippingPledgeViewModelType, NoShippin
         paymentSourceId: paymentSourceId,
         setupIntentClientSecret: nil,
         applePayParams: applePayParams,
-        refTag: refTag
+        refTag: refTag,
+        incremental: false // TODO: implementation in [mbl-1853](https://kickstarter.atlassian.net/browse/MBL-1853)
```

Comment:
It looks like `incremental` is always false in this PR, is the idea to make it true sometimes in that ticket?